gslab_misc/ado/stata.toc

```diff
@@ -0,0 +1,34 @@
+p aivreg                Estimates panel IV models using 2SLS within estimator
```

Comment:
- I added author attributions
- All of the ado files except two either had no authorship label or had help files lising MG, JMS, and/or RAs as authors. Those two are:
    - matrix_to_txt.ado: by  Michael Blasnik (M. Blasnik & Associates, Boston, MA), Ben Jann (ETH Zurich), and Matthew Gentzkow
    - placevar.ado: by Jeroen Weesie (Dept of Sociology/ICS, Utrecht University)